### PR TITLE
Awesome books [Microverse Project]

### PR DESCRIPTION
**Pull request by mr-abdellah**
In this project, we create books dynamically when the user enters values into inputs and taps the 'Add' button, and delete the specific book when the user tap the 'Delete' button

### CODE REVIEW

**Review by omar-muhamad — COMMENTED** [review]
# Invalid ❌

Hi @mr-abdellah,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Your Code Review Request is marked as invalid for these reasons:

- [ ] You didn't document your work professionally:
   - You forgot to edit your README file.
- [ ] Some linters checks are not passed. 


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

File: script.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+const addBtn = document.getElementById('add');
+
+addBtn.addEventListener('click', () => {
+  const newBook = Object.create(Book.prototype);
+
+  newBook.title = title.value;
+  newBook.author = author.value;
+
+  window.localStorage.setItem('Title', title.value);
```
Comment (SajjadAhmad14) [inline script.js]: The user is unable to see books added after reloading the page. Please make sure the data is preserved with local storage and does not disappear after page reload.

File: README.md [hunk 2]
```diff
@@ -1 +1,47 @@
-# Awesome-Book
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awesome Books
+
+In this project, we built a basic website that allows users to add/remove books from a list. This was achieve by using JavaScript objects, arrays, modifying the DOM dynamically and adding basic events Listeners.
+
+## Live Demo
+
```
Comment (SajjadAhmad14) [inline README.md:8]: Please add instructions to set up the project locally as it is necessary for the non-technical user of your project.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi team, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
PR has a title and summary. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by zainsadaqat — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey @mr-abdellah   😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**